demos/common/python/helpers.py

```diff
@@ -51,3 +50,8 @@ def log_runtime_settings(exec_net, devices):
             except RuntimeError:
                 pass
     log.info('\tNumber of network infer requests: {}'.format(len(exec_net.requests)))
+
+def log_latency_per_stage(*metrics):
+    names = ('Decoding', 'Preprocessing', 'Inference', 'Postprocessing', 'Rendering')
```

Comment:
It would be better, if you pass to the function dictionary `name: metric`

Reply:
Updated

Reply:
Iterating though dict doesn't guarantee the order will stay the same as it was for arguments of dict constructor thus it can't be a dict.

C++ `printStagesLatency()` takes doubles, so you should take separate vars here, or ask @ivikhrev to update it as well.